webview/src/experiments/components/table/styles.module.scss

```diff
@@ -258,11 +258,14 @@ $bullet-size: calc(var(--design-unit) * 4px);
 
   .tableContainer {
     flex: 1;
+    display: flex;
+    align-items: stretch;
   }
 
   table {
     display: inline-block;
     border-collapse: collapse;
+    flex-grow: 1;
```

Comment:
I fixed the issue by making the `table` element the width/height of the webview screen. Everything appears to be working correctly, but are there any issues that anyone else spots with this solution?